.github/workflows/coding.yml

```
@@ -0,0 +1,58 @@
+name: Coding Test
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)